ftplugin/nix.vim

```diff
@@ -19,3 +19,116 @@ if get(g:, 'nix_recommended_style', 1)
     \ softtabstop=2
     \ expandtab 
 endif
+
+" Borrowed from vim-markdown: https://github.com/plasticboy/vim-markdown/
+if exists('g:vim_nix_fenced_languages')
+    let s:filetype_dict = {}
+    for s:filetype in g:vim_markdown_fenced_languages
```

Comment:
This should also be `vim_nix_fenced_languages`.

Reply:
Whoops.  Thanks, I somehow missed that :sweat_smile: